### PR TITLE
add description of stale read metrics

### PR DESCRIPTION
**Pull request by you06**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

TiDB v6.5.3 introduces two metrics for Stale Read.
 
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): pingcap/docs#13842, https://github.com/pingcap/tidb/issues/43325, https://github.com/pingcap/tidb/pull/43324

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: grafana-tidb-dashboard.md [hunk 1]
```diff
@@ -112,6 +112,8 @@ aliases: ['/docs-cn/dev/grafana-tidb-dashboard/','/docs-cn/dev/reference/key-mon
 - KV Request OPS：KV Request 根据 TiKV 显示执行次数
 - KV Request Duration 99 by store：根据 TiKV 显示 KV Request 执行时间
 - KV Request Duration 99 by type：根据类型显示 KV Request 的执行时间
+- Stale Read OPS: 每秒执行的 Stale Read 的数量，根据结果分为 hit 和 miss 进行统计
+- Stale Read Traffic：Stale Read 消耗的流量，根据结果分为 hit 和 miss 进行统计
```
Comment (Oreoxmt) [inline grafana-tidb-dashboard.md:116]:
```suggestion
- Stale Read OPS：每秒执行的 Stale Read 的数量，根据结果分为 hit 和 miss 进行统计
- Stale Read Traffic：Stale Read 消耗的流量，根据结果分为 hit 和 miss 进行统计
```

**Review by Oreoxmt — APPROVED** [review]

**Review by qiancai — APPROVED** [review]
/lgtm

**Pull request merged**